### ISSUE TITLE
Multiple camera / settings rendering

### ISSUE THREAD

**Issue by subsoap** [rgrams/rendercam#41]
Sometimes there are situations where you want to be able to render multiple layers of a scene with different camera setups. For example, in a FPS you might want to render the world at 90 FOV but the first person model at 45 or 70 FOV.

Besides requiring user editing the render script, could this be an added feature for consideration somehow?

**Comment by rgrams** [comment]
Yeah, I really want this feature too. But I also want to fix the issue with the camera lagging by 1 frame. Unfortunately, without using a weird hacks or a native extension, there's no way to have both (as discussed in [this thread](https://forum.defold.com/t/getting-an-up-to-date-camera-transform/65068)). 

That said, hardly anyone has complained about the 1-frame lag so far, so it will just be that one camera is up-to-date and any other cameras will lag.

**Comment by subsoap** [comment]
For a camera setup like FPS it may be possible to do with a single camera but changing the FOV (or other settings) between render predicate steps.
